oam/install_scripts/mcs-loadbrm.py.in

```diff
@@ -106,20 +106,25 @@ file=sys.stderr)
                     print("Pulling {} from the primary node.".format(e))
                     url = "https://{}:{}/cmapi/{}/node/meta/{}".format(primary_address, \
 api_port, api_version, e)
-                    r = requests.get(url, verify=False, headers=headers, timeout=2)
+                    r = requests.get(url, verify=False, headers=headers, timeout=30)
                     if (r.status_code != 200):
                         raise RuntimeError("Error requesting {} from the primary \
 node.".format(e))
 
                     # To avoid SM storing BRM files
                     if storage.lower() == 's3' and bucket.lower() != 'some_bucket':
                         dbrmroot = BYPASS_SM_PATH
+
+                        if not os.path.exists(dbrmroot):
+                            os.makedirs(dbrmroot)
```

Comment:
This is a change for our docker + sharedstorage solution. BYPASS_SM_PATH does not always exists when in a sharedstorage setting, Let's make sure to create it if we're going to use it.